infrastructure/tasks/00-cosmosdb_setup.ts

```diff
@@ -146,8 +146,10 @@ export const run = async () => {
   );
 };
 
-run()
+checkEnvironment()
+  .then(() => readConfig(process.env.ENVIRONMENT))
```

Comment:
this script must be way much more verbose telling exactly what is going on, whenever you touch infrastructure stuff anything can go wrong and you need to know exactly what has been done and in what state the infrastructure is